vm/adb/adb.go

```diff
@@ -110,7 +110,7 @@ func (pool *Pool) Create(workdir string, index int) (vmimpl.Instance, error) {
 		}
 	}
 	// Remove temp files from previous runs.
-	if _, err := inst.adb("shell", "rm -Rf /data/syzkaller*"); err != nil {
+	if _, err := inst.adb("shell", "rm -Rf /data/syz-*"); err != nil {
```

Comment:
This is supposed to delete temp dirs like "syzkaller-testdir".
We don't need to delete binaries because we will overwrite them anyway.

Reply:
oh, sorry, misunderstood